### DISCUSSION TITLE
Saving Youtube / Media links

### DISCUSSION THREAD

**Discussion by abdessalaam** [wallabag/wallabag#7075] [Ideas]
I was wondering if there'd be a way to implement saving links to YouTube videos and other media...? At the moment when I try to save a YT link it saves consent.youtube.com. The link to the original video is still clickable in the header, but the whole page is just legal jargon...

A couple of reasons why I'd like to save video links in Wallabag:
- to have all my references in one place (rather than going to saved playlists on YT specifically)
- some videos (like "made for kids") don't allow saving on YT
- to find and index them easier
- to share and quickly access all saved resources across different devices
- because Wallabag is just awesome :)

Probably media / videos would be saved as a Title, preview thumbnail, short description (or perhaps the whole description?), link and tags... so not really parsed in the same way as text pages. That's my guess.

**Comment by j0k3r** [comment]
Fetching YouTube links should be fixed in the next release (the fix: https://github.com/fivefilters/ftr-site-config/pull/1285)

**Reply by abdessalaam** [reply]
Amazing, thanks!

**Reply by abdessalaam** [reply]
Just upgraded - works beautifully.
